src/test/java/org/selenium/pages/tests/LogInPageTest.java

```diff
@@ -0,0 +1,35 @@
+package org.selenium.pages.tests;
+
+import io.qameta.allure.Epic;
+import io.qameta.allure.Step;
+import jdk.jfr.Description;
+import org.junit.jupiter.api.AfterEach;
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.Test;
+import org.junit.jupiter.api.extension.ExtendWith;
+import org.selenium.utils.ListenerEvent;
+
+import static org.junit.Assert.assertEquals;
+
+@ExtendWith(ListenerEvent.class)
+public class LogInPageTest extends BaseTest {
+    @BeforeEach
+    @Step("Beginning the action")
+    public void getStart() {
+        mainPage.logIn();
```

Comment:
Please save the result of the method in page objects that you created in BaseTest
Like
signIn= mainPage.logIn();